.travis.yml

```diff
@@ -108,7 +108,26 @@ env:
       notification/mail.py
       notification/mqtt.py
       notification/sns.py
-      notification/typetalk.py"
+      notification/typetalk.py
+      storage/netapp_e/na_eseries_ethernet.py
+      storage/netapp_e/na_eseries_hostgroup.py
+      storage/netapp_e/na_eseries_amg_sync.py
+      storage/netapp_e/na_eseries_amg.py
+      storage/netapp_e/na_eseries_consistency_group.py
+      storage/netapp_e/na_eseries_snapshot_group.py
+      storage/netapp_e/na_eseries_snapshot_volume.py
+      storage/netapp_e/na_eseries_host.py
+      storage/netapp_e/na_eseries_facts.py
+      storage/netapp_e/netapp_storage_system.py
+      storage/netapp_e/na_eseries_volume.py
+      storage/netapp_e/na_eseries_snapshot_images.py
+      storage/netapp_e/na_eseries_lun_mapping.py
+      storage/netapp_e/na_eseries_flashcache.py
+      storage/netapp_e/na_eseries_controller_firmware.py
+      storage/netapp_e/na_eseries_storagepool.py
+      storage/netapp_e/na_eseries_auto_upgrade.py
+      storage/netapp_e/na_eseries_auth.py
+      storage/netapp_e/na_eseries_volume_copy.py"
```

Comment:
I was just trying to get past the CI checks. :)  Thanks for the context .  I don't think we have any (but I didn't write these modules myself) dependencies that necessitate 2.4 or 3.x blacklisting.  I returned this file to it's original state prior to me blacklisting the modules in the queue.  This change is staged for my next push.  Awaiting feedback from @bcoca  on an error handling proposal; will push then.